app/controllers/users_controller.rb

```diff
@@ -4,6 +4,12 @@ def index
   end
 
   def show
-    @user = User.find(params[:id])
+    @user = User.find_by_id(params[:id])
+    if @user
+      @recent_posts = @user.most_recent_posts
+    else
+      flash[:alert] = 'User not found'
+      redirect_to root_path
+    end
```

Comment:
- [ ] Excellent work! 👏🏻 However, in the User show page, as depicted in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png), the total number of posts should be 4, but only 2 posts are currently displayed. Additionally, when clicking on the [See all posts](http://localhost:3000/users/11/posts) button, directing me to the posts index page to view all posts, only three of them are being displayed. Kindly ensure these values align correctly.

![image](https://github.com/firstpj/blog-app/assets/79058364/6784560b-8457-4eae-9f92-1c472d36d7a9)